package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "suomifi-design-tokens",
-  "version": "3.2.0",
+  "version": "4.2.0",
```

Comment:
As it is new major we would want to start from beginning, so it should be `4.0.0`.

Reply:
New PR was created to solve also this issue #55